src/lib/viewers/media/MP3.scss

```diff
@@ -25,6 +25,7 @@
     .bp-media-gear-icon {
         position: absolute;
         right: 10px;
+        width: 36px;
```

Comment:
Is this change intentional?

Reply:
Yes, because the `SettingsToggle` no longer uses the `bp-MediaButton` mixin, so the width needs to be adjusted